### ISSUE TITLE
Disk usage not working in prod

### ISSUE THREAD

**Issue by AndrewJGaut** [codalab/codalab-worksheets#4472]
Disk usage is not working in production; users' disk is not being accurately computed when deleting/creating bundles. for instance, see [here](https://github.com/codalab/codalab-worksheets/issues/4469).

It would appear the changes I made in [this PR](https://github.com/codalab/codalab-worksheets/pull/4354) are not working properly in production.

Luckily, I can revert back to the old method of computing disk quota and it will recompute their quota (see [here](https://github.com/codalab/codalab-worksheets/blob/7fc08ccda2d0c6a0208be8c327efbedf3ae7a53d/codalab/model/bundle_model.py#L1132-L1156) and, in particular, `update_user_disk_used` [here](https://github.com/codalab/codalab-worksheets/blob/7fc08ccda2d0c6a0208be8c327efbedf3ae7a53d/codalab/model/bundle_model.py#L2776-L2780), which uses a query [here](https://github.com/codalab/codalab-worksheets/blob/7fc08ccda2d0c6a0208be8c327efbedf3ae7a53d/codalab/model/bundle_model.py#L2762-L2769) to just recompute the user disk quota from scratch).

In short, I can effectively revert this change for the time being and the user's disk usage will automatically be reset to their true usage whenever they next make any changes that affect disk quota (e.g. upload a bundle or delete a bundle). I will investigate why this change passed the test and seems to work sometimes (e.g. when I tested it in dev) but not at other times. I suspect there may be a race condition somewhere.
